securesignals-gma-dev-app/src/main/java/com/uid2/dev/utils/BundleEx.kt

```diff
@@ -0,0 +1,7 @@
+package com.uid2.dev.utils
+
+import android.os.Bundle
+
+private const val UID2_ENVIRONMENT_EUID = "uid2_environment_euid"
+
+fun Bundle.isEnvironmentEUID(): Boolean = getBoolean(UID2_ENVIRONMENT_EUID, false)
```

Comment:
I see what you're doing here, but I think it's better just to fold the three functions into 1: 

```
internal fun Context.isEnvironmentEUID(): Boolean
```